### PR TITLE
Misc build-sys cleanups/improvements

### PR DESCRIPTION
**Pull request by cfergeau**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,7 @@
 PREFIX=/go
 CMD=crc-driver-libvirt
 DESCRIBE=$(shell git describe --tags)
+CONTAINER_RUNTIME ?= podman
```
Comment (praveenkumar) [inline Makefile:4]: This is the exactly I had in mind , but I think default should be `docker` here. :+1:

Comment (praveenkumar) [reply Makefile:4]: Or add another commit to change the circle CI config and have this env set to `docker`

Comment (cfergeau) [reply Makefile:4]: > Or add another commit to change the circle CI config and have this env set to docker

Testing this, but I may also have to rename back the images from Containerfile to Dockerfile

File: .circleci/config.yml [hunk 2]
```diff
@@ -13,11 +13,11 @@ jobs:
...
+       command: make CONTAINER_RUNTIME="docker"
     - store_artifacts:
-       path: ~/work/crc-driver-libvirt-centos7
+       path: ~/work/crc-driver-libvirt-centos8
        destination: crc-driver-libvirt
     - store_artifacts:
-       path: ~/work/crc-driver-libvirt-ubuntu16.04
-       destination: crc-driver-libvirt-ubuntu16.04
+       path: ~/work/crc-driver-libvirt-ubuntu20.04
+       destination: crc-driver-libvirt-ubuntu20.04
```
Comment (praveenkumar) [inline .circleci/config.yml:22]: :+1:

**Review by praveenkumar — APPROVED** [review]

**Pull request merged**